src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAOSpec.scala

```diff
@@ -38,6 +39,61 @@ class GraphWorkspaceDAOSpec extends FlatSpec with Matchers with OrientDbTestFixt
     }
   }
 
+  it should "save a new workspace with a realm" in withDefaultTestDatabase { dataSource =>
+    dataSource.inTransaction() { txn =>
+      val user = RawlsUser(RawlsUserSubjectId("user@secret.com"), RawlsUserEmail("user@secret.com"))
+      authDAO.saveUser(user, txn)
+      val secretGroup = makeRawlsGroup("SecretRealm", Set(user), Set.empty)
+      authDAO.saveGroup(secretGroup, txn)
+      val secretWorkspace = Workspace(
+        namespace = testData.wsName.namespace,
+        name = testData.wsName.name,
+        realm = Some(secretGroup),
+        workspaceId = "aWorkspaceId",
+        bucketName = "aBucket",
+        createdDate = testDate,
+        lastModified = testDate,
+        createdBy = "Secret User",
+        attributes = Map("workspace_attrib" -> AttributeString("foo")),
+        accessLevels = Map.empty
+      )
+
+      workspaceDAO.save(secretWorkspace, txn)
+      // now check explicitly that the vertex exists. note that this will fail if our reserved keywords change.
+      assert {
```

Comment:
I would actually load it by calling workspaceDAO.findById and make assert that what is loaded is the same as what was saved.